.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: test
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  integration:
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x]
+        criu-version: [3.15]
+        systemd: ["systemd", ""]
+        rootless: ["rootless", ""]
+    runs-on: ubuntu-20.04
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - name: install deps
+      run: |
+        sudo apt update
+        sudo apt install libseccomp-dev
+    - name: install criu
+      env:
+        VERSION: 3.15
+      run: |
+        mkdir criu
+        curl -fsSL https://github.com/checkpoint-restore/criu/archive/${{ matrix.criu-version }}.tar.gz | tar -C criu -xz --strip-components=1
```

Comment:
`v` is missing before `${{ matrix.criu-version }}`.

Reply:
thanks, fixed! this is still a WIP as I am trying to make it work